src/mpyl/project.py

```diff
@@ -134,12 +129,12 @@ class StageSpecificProperty(Generic[T]):
     deploy: Optional[T]
     postdeploy: Optional[T]
 
-    def for_stage(self, stage: Stage) -> Optional[T]:
-        if stage == Stage.BUILD:
+    def for_stage(self, stage: str) -> Optional[T]:
+        if stage == "build":
             return self.build
-        if stage == Stage.TEST:
+        if stage == "test":
             return self.test
-        if stage == Stage.DEPLOY:
+        if stage == "deploy":
             return self.deploy
         return self.postdeploy
```

Comment:
Shouldn't we throw instead of returning postdeploy as default in case a stage gets passed that isn't implemented (yet)?